packages/angular_devkit/core/src/logger/transform-logger_spec.ts

```diff
@@ -6,21 +6,19 @@
  * found in the LICENSE file at https://angular.io/license
  */
 // tslint:disable:no-any
-import 'rxjs/add/operator/filter';
-import 'rxjs/add/operator/map';
+import { filter, map, toArray } from 'rxjs/operators';
```

Comment:
These should be separate imports:

import { filter } from 'rxjs/operators/filter';
...

Reply:
Per discussion in https://github.com/angular/angular-cli/pull/8976#discussion_r158592854, this is a non-starter/not necessary